docs/specification/STU2.md

```diff
@@ -6,13 +6,13 @@
 ## Overview
 The FHIRcast specification describes the APIs used to synchronize disparate healthcare applications' user interfaces in real time,  allowing them to show the same clinical content to a user (or group of users). 
 
-Once the subscribing app [knows about the session](#session-discovery), the app may [subscribe](#subscribing-and-unsubscribing) to specific workflow-related events for the given session. The subscription is [verified](#intent-verification-request) and the app is [notified](#event-notification) when those workflow-related events occur; for example, by the clinician opening a patient's chart. The subscribing app may [initiate context changes](#request-context-change) by accessing APIs exposed by the Hub; for example, closing the patient's chart. The app [deletes its subscription](#unsubscribe) to no longer receive notifications. The notification message describing the workflow event is a simple json wrapper around one or more FHIR resources. 
+Once the subscribing app [knows about the session](#session-discovery), the app may [subscribe](#subscribing-and-unsubscribing) to specific workflow-related events for the given session. The app is [notified](#event-notification) when those workflow-related events occur; for example, by the clinician opening a patient's chart. The subscribing app may [initiate context changes](#request-context-change) by accessing APIs; for example, closing the patient's chart. The app [deletes its subscription](#unsubscribe) to no longer receive notifications. The notification message describing the workflow event is a simple json wrapper around one or more FHIR resources. 
```

Comment:
For issue 25878 seems like we just removed the "by the Hub" part, but didn't define the Hub. Should we?

Reply:
Hey Will,

I was considering this new sentence as a definition of the Hub:
>A Hub exposes APIs for subsubscribing and unsubscribing, requesting context changes and also distribute event notifications. Hubs SHOULD support WebSockets and MAY support webhooks.

I considered WebSub's definition as a model: https://www.w3.org/TR/websub/#hub

What do you think?

Isaac

Reply:
Yeah that looks perfect to me.